animation_nodes/ui/node_menu.py

```diff
@@ -303,6 +310,15 @@ def draw(self, context):
             if dataType not in mainBaseDataTypes:
                 insertNode(layout, "an_CombineListsNode", dataType, {"assignedType" : repr(dataType)})
 
+class MixListMenu(bpy.types.Menu):
```

Comment:
Perhaps we should add every type to its category similar to the *Mix* node. Do you agree?

Reply:
Most of these menus are already so cluttered, what do you think about adding a list submenu for all of them?

Reply:
I will ask Jacques and get back to you.

Reply:
You should add those nodes to the `AnimationMenu`. Just add a separator and add the nodes. You may also automate the creation of the Animate nodes using `numericalDataTypes` just as you did here.

Reply:
Done in 3652c9a.